src/transformers/models/led/modeling_led.py

```diff
@@ -160,6 +160,8 @@ def __init__(self, config, layer_id):
 
         self.one_sided_attn_window_size = attention_window // 2
 
+        self.config = config
```

Comment:
Out of curiosity, why are we changing `modeling_led.py` in this PR?

Reply:
It's the automatic `# Copied from ...` script that does that - seems ok to me!

Reply:
Exactly 😊 I think it should be pretty straightforward to add ONNX support for LED after this.